src/tests/esearch.mbox

```diff
@@ -1,28 +1,42 @@
-From user@domain  Fri Feb 22 17:06:21 2008
+From user@domain  Fri Feb 22 17:06:23 2008
```

Comment:
Why did you change the timestamps? I guess it doesn't matter since the tests aren't actually using timestamp searches.

Reply:
No reason, just copy/paste. Can revert to older if you want?

Reply:
Would be nice to avoid unnecessary changes, causing confusion in history.

Reply:
Done